<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="4">
     <widget class="QLineEdit" name="lineEdit5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QLineEdit" name="lineEdit6">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QLineEdit" name="lineEdit7">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="8">
     <widget class="QLineEdit" name="lineEdit8">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="openMyComBtn">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="closeMyComBtn">
      <property name="text">
       <string>关闭串口</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit9">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(206, 206, 206);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit10">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>串口接收的数据：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="4" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>串口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>比特率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数据位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>校验位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>停止位：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="portNameComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="baudRateComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="dataBitsComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="parityComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="stopBitsComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTextBrowser" name="textBrowser1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="lineEdit1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QTextBrowser" name="textBrowser2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6" colspan="2">
     <widget class="QPushButton" name="ClearButton">
      <property name="text">
       <string>清除数据</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="lineEdit3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="lineEdit4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);
</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>分割后的字符</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(207, 207, 207);</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
